### PR TITLE
[DOCFIX] update File-System-API of chinese version

### PR DESCRIPTION
**Pull request by njuxiaoqian**

### CODE REVIEW

**Comment by ifcharming** [comment]
ok to test

File: docs/cn/File-System-API.md [hunk 1]
```diff
@@ -83,7 +87,7 @@ Alluxio提供定位策略，用于确定应该选择哪个Worker来存储文件
 
     返回指定主机名的Worker。该策略不能被设置为默认策略。
 
-Alluxio支持自定义策略，所以你可以通过实现`alluxio.client.file.policyFileWriteLocationPolicy`接口来开发定制的定位策略来自己的迎合应用需求。注意默认策略必须要有一个空构造函数。要想使用ASYNC_THROUGH写类型，所有的文件数据块必须被写到相同的Worker上。
+Alluxio支持自定义策略，所以你可以开发自己的定位策略来迎合应用需求通过实现接口`alluxio.client.file.policyFileWriteLocationPolicy`。注意默认策略必须要有一个空构造函数。要想使用ASYNC_THROUGH写类型，所有的文件数据块必须被写到相同的Worker上。
```
Comment (ifcharming) [inline docs/cn/File-System-API.md]: nit: 所以你可以通过实现接口`alluxio.client.file.policyFileWriteLocationPolicy`，开发自己的定位策略来迎合应用需求。

Does this sound better?

**Comment by ifcharming** [comment]
Thank you for the PR! Overall looks great.
Left a minor comment.

File: docs/cn/File-System-API.md (hunk 1 above)
Comment (njuxiaoqian) [inline docs/cn/File-System-API.md]: @ifcharming Thanks for review! Updated!

**Comment by yupeng9** [comment]
LGTM. Thanks.

**Pull request merged**